Bottom: 7413d85582079678a2245ca49e793f0cd8ad5bd2
Top:    b41f5d11d3f72c1bbe451336c858f7ebfb0a4da7
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-31 10:19:36 -0700

Refresh of addrinfo-af-ib

---

diff --git a/examples/cmatose.c b/examples/cmatose.c
index ba4fc72..94fc4cd 100644
--- a/examples/cmatose.c
+++ b/examples/cmatose.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005-2006,2011-2012 Intel Corporation.  All rights reserved.
+ * Copyright (c) 2005-2006,2011 Intel Corporation.  All rights reserved.
  *
  * This software is available to you under a choice of one of two
  * licenses.  You may choose to be licensed under the terms of the GNU
